src/components/HelpDialog.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <dialog-base
+    :visible.sync="isVisible"
+    :title="t('helpDialog.title')"
+  >
+    <div class="help-block help-links">
+      <div v-for="(link, index) in links" :key="index">
+        <a
+          class="help-links-item"
+          :href="link.url"
+          target="_blank"
+        >
+          <span>{{ link.title }}</span>
+          <s-icon name="external-link" size="16" />
+        </a>
+        <s-divider v-if="index !== links.length - 1" class="help-links-item_divider" />
+      </div>
+    </div>
+    <div class="help-information">
+      <div class="help-block">
+        {{ t('helpDialog.appVersion') }} {{ appVersion }}
+      </div>
+      <div class="help-block">
+        {{ t('helpDialog.contactUs') }}<br>
+        polkaswap@soramitsu.co.jp
```

Comment:
Maybe we should move email to config somehow. It's a bit strange to have it as text.